### ISSUE TITLE
NVDA is not announcing the 'Wrap type' for the Illustrations in Word Document.

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#9399]
### Steps to reproduce:
1. Open a word document
2. Insert some illustration in that (Picture/SHape/Online Picture/Textbox/SmartArt/Chart etc)
3. Select the illustration and change text wrapping for those to "Inline with Text"/Tight/Square/Infront of Text/Behing text etc from "Wrap Text" menu in related format tab.
4. Select the illustration
5. Observe that NVDA is not announcing the Wrap type for those objects.

### Actual behavior:
NVDA is not announcing the wrap type while navigating to the illustrations using either Tab key/ Arrow key 

### Expected behavior:
NVDA should announce the wrap type(Inline with Text, In Front of Text, etc.) while navigating to the illustrations.

#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
No

**Comment by Adriani90** [comment]
The implementation of this will depend on #5599.
